uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -437,6 +441,156 @@ impl PythonCodeOracle {
     }
 }
 
+impl VisitMut for PythonCodeOracle {
+    fn visit_record(&self, ci: &mut ComponentInterface) {
+        // Conversions for RecordTemplate.py
+        let mut new_records: BTreeMap<String, Record> = BTreeMap::new();
+
+        for (key, record_item) in ci.records.iter_mut() {
+            let mut record = record_item.clone();
+
+            // We just want to prefix reserved keywords for the name, without modifying it
+            record.rename(self.class_name(record_item.name()));
+
+            for field in &mut record.fields {
+                field.rename(self.var_name(field.name()));
+            }
+
+            new_records.insert(key.to_string(), record);
```

Comment:
Maybe key should be transformed as well?  I think this goes with the suggestion to transform the `Type` instances.